src/snippets/reviewed-preprint.v1.yaml

```diff
@@ -83,6 +83,12 @@ properties:
                 - thumbnail
             - required:
                 - social
+    elifeAssessment:
+        type: object
+        $ref: ../misc/elife-assessment-terms.v1.yaml
+        required:
```

Comment:
The set of required properties on an `elifeAssessment` is inconsistent depending on whether it's a snippet of an `article-vor` or a snippet of a `reviewed-preprint`. This can be confusing when we model these in PHP code, as many properties can be null

If the `elifeAssessment` properties are not present on the API responses yet, could we consider a stricter contract so that they appear in an atomic change?

Reply:
e.g. `title` and `content` required, `significance` and `strength` optional for the time being

Reply:
We have reviewed this and strive for a single elifeAssessment schema which should have all properties as optional, in order to implement it easily from the producers. From the point of view of journal this would be a single php class.